db/migrate/20230117190802_create_workflow_migrations.rb

```diff
@@ -0,0 +1,31 @@
+class CreateWorkflowMigrations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :workflows do |t|
+      t.references :ems,           :type => :bigint, :index => true, :references => :ext_management_system
+      t.references :tenant,        :type => :bigint, :index => true
+      t.references :git_reference, :type => :bigint, :index => true
+      t.string :type, :index => true
+      t.string :name
+      t.jsonb :payload, :default => {}
+      t.jsonb :credentials, :default => {}
+
+      t.timestamps
+    end
+
+    create_table :workflow_instances do |t|
+      t.references :ems,      :type => :bigint, :index => true, :references => :ext_management_system
+      t.references :workflow, :type => :bigint, :index => true
+      t.references :tenant,   :type => :bigint, :index => true
```

Comment:
The current code base is half way between group ownership and tenant ownership. Could we add miq group if here as well? (All tables)

Reply:
:+1: great point I will add